po/en_US.po

```diff
@@ -6571,6 +6571,18 @@ msgstr ""
 msgid "Running on:"
 msgstr ""
 
+#: util.cpp:1495
```

Comment:
There is no need to change this file. It changes automatically. You need to add new phrases only to the mmex.pot file.